gltf/write.cpp

```diff
@@ -739,6 +739,14 @@ static std::string decodeUri(const char* uri)
 
 void writeImage(std::string& json, std::vector<BufferView>& views, const cgltf_image& image, const ImageInfo& info, size_t index, const char* input_path, const char* output_path, const Settings& settings)
 {
+	if (image.name)
```

Comment:
Is preserving the name of the image object necessary? It doesn't carry semantical meaning, and gltfpack right now only preserves names for objects that are likely to be used by-name externally, specifically nodes, materials and animations.

Reply:
I need to check if this is required by the internal material definition called MTOON.